templates/linux/squeak.sh

```diff
@@ -93,9 +133,24 @@ ensure_image() {
   fi
 }
 
+detect_sound() {
+    if pulseaudio --check 2>/dev/null ; then
+        if "$VM" --help 2>/dev/null | grep -q vm-sound-pulse ; then
+	    VMOPTIONS="$VMOPTIONS -vm-sound-pulse"
+        else
+            VMOPTIONS="$VMOPTIONS -vm-sound-oss"
+            if padsp true 2>/dev/null; then
+                SOUNDSERVER=padsp
+            fi
+        fi
+    fi
+}
+
 ensure_kernel
+ensure_cpu
 ensure_vm
 ensure_image
+detect_sound
 
-echo "Using ${VM}..."
-exec "${VM}" "${IMAGE}"
+set -x
```

Comment:
Do we really want the next line to be printed? Or is this just an oversight/leftover?

Reply:
Yes. the line is printed so that we know exactly what options are used for the launch

Reply:
this must not end up in the shipped vm. 
Please remove it :)

Reply:
The version in shipped VM had
  echo "Using ${VM} ..." 
which did not reveal the exact options being used. Would you like me to restore that version?